listenbrainz/webserver/views/test/test_index.py

```diff
@@ -173,9 +167,9 @@ def test_menu_logged_in_error_dont_show_user_loaded(self, mock_user_get):
 
         mock_user_get.return_value = user
 
-        @self.app.route('/page_that_returns_500')
+        @self.app.route('/page_that_returns_500_2')
```

Comment:
was this for testing, or is it required?

Reply:
It is required because the flask app is same for all tests in this class and another method registers a view of the same name.